values.yaml

```diff
@@ -36,7 +36,7 @@ cluster:
   # join: node1,node2,node3
   # or
   # join: 10.1.5.07,10.1.5.08,10.1.5.09
-  join: 
+  join: 172.20.32.153,172.20.52.242,172.20.62.33
```

Comment:
Would be great to remove this from the commit by squashing the two commits.

Reply:
Fixed.